cmd/root.go

```diff
@@ -188,6 +202,9 @@ func init() {
 	rootCmd.Flags().BoolVarP(&opt.TTY, "tty", "t", opt.TTY, "Stdin is a TTY")
 	rootCmd.Flags().StringSliceVar(&opt.Includes, "include", []string{}, "Include only specific paths from current directory for syncing")
 	rootCmd.Flags().StringSliceVar(&opt.Excludes, "exclude", []string{}, "Exclude only specific paths from current directory for syncing")
+	rootCmd.Flags().StringVar(&opt.ServiceAccountName, "service-account-name", opt.ServiceAccountName, "The service account name that you want the pod to use")
+	rootCmd.Flags().StringSliceVar(&opt.NodeSelectorKeys, "node-selector-keys", []string{}, "The node selector keys that you want applied")
+	rootCmd.Flags().StringSliceVar(&opt.NodeSelectorValues, "node-selector-values", []string{}, "The node selector values that you want applied")
```

Comment:
Can you use `StringToString` flag? https://godoc.org/github.com/spf13/pflag#StringToString

Reply:
Thanks, I was looking for a function like this, I don't know how I missed it.